Doc/fix link and edit (#78)
pyansys/pypim: doc/source/integration.rst

   mapdl = launch_mapdl(exec_file="/usr/bin/mapdl")

Start a gRPC product
====================

To use PyPIM, the code flow should first check if PyPIM is configured

> with the :func:`is_configured` method and then check to ensure that you have not

Author: @plule-ansys I'm basicallu suggesting we put it back like you  had it it! I totally understand your point.

> with the :func:`is_configured` method and then check to ensure that you have not

Author: ```suggestion
to use the :func:`is_configured` method and then check to ensure that the user has not
```

> with the :func:`is_configured` method and then check to ensure that you have not

Author: ```suggestion
to use the :func:`is_configured` method and then check to ensure that the user has not
```
